sdlib/d/gc/region.d

```diff
@@ -234,25 +254,43 @@ unittest acquire_release {
 	void* addr0;
 	assert(regionAllocator.acquire(&addr0));
 
+	// check the block counter works
```

Comment:
Capital and `.`.

But in general, this is either superfluous or would better fit as an error message in the assert.